### DISCUSSION TITLE
The /miner/kvlog/metadata/.log.cbor files are occupying a lot of disk space

### DISCUSSION THREAD

**Discussion by Pacificwestcoast** [filecoin-project/lotus#6615] [Storage Provider] [answered]
Hello,

I am running calibnet v13 for just 5 days,
lotus version 1.10.0-rc6+calibnet+git.a641a0390

Today I got warning said my 500GB system hard disk almost full, I checked that this file takes  445GB, /miner/kvlog/metadata/***.log.cbor.
how could I do?

Thanks.

**Comment by dd45e640b42e6da7da96faee3996ef7c** [comment, marked as answer]
you can turn of the meta data log in the config.toml and delete the log file
